frontend/www/js/omegaup/arena/contest_contestant.ts

```diff
@@ -33,7 +33,7 @@ OmegaUp.on('ready', () => {
   time.setSugarLocale();
   const payload = types.payloadParsers.ContestDetailsPayload();
   const commonPayload = types.payloadParsers.CommonPayload();
-  const contestAdmin = payload.adminPayload?.contestAdmin ?? false;
+  const contestAdmin = payload.adminPayload;
```

Comment:
```suggestion
  const contestAdmin = Boolean(payload.adminPayload);
```

así nos aseguramos que el tipo sea el correcto

Reply:
Listo.